src/org/openlcb/cdi/swing/CdiPanel.java

```diff
@@ -250,7 +251,7 @@ DisplayPane createGroupPane(CdiRep.Group item, long origin, int space) {
             }
             JPanel currentPane = this;
             JTabbedPane tabbedPane = new JTabbedPane();
-            currentPane.add(tabbedPane);
+            tabbedPane.setTabLayoutPolicy(JTabbedPane.SCROLL_TAB_LAYOUT);
```

Comment:
That was part of an attempt to fix the scrolling alignment issue, and it didn't work.  I'm ok taking that out.